src/actions/AddProblemOperatorAction.C

```diff
@@ -0,0 +1,25 @@
+#include "AddProblemOperatorAction.h"
+
+registerMooseAction("PlatypusApp", AddProblemOperatorAction, "add_mfem_problem_operator");
+
+InputParameters
+AddProblemOperatorAction::validParams()
+{
+  InputParameters params = Action::validParams();
+  params.addClassDescription(
+      "Set the ProblemOperator used in the MFEMExecutioner to solve the FE problem.");
+  return params;
+}
+
+AddProblemOperatorAction::AddProblemOperatorAction(const InputParameters & parameters)
+  : Action(parameters)
+{
+}
+
+void
+AddProblemOperatorAction::act()
+{
+  MFEMProblem * mfem_problem = dynamic_cast<MFEMProblem *>(_problem.get());
+  if (mfem_problem)
+    mfem_problem->setProblemOperator();
```

Comment:
Would `initProblemOperator()` be a better name for this method? `set` normally implies you're providing the value to set it to?

Reply:
Yes - as we currently don't support adding custom ProblemOperators, initProblemOperator() is a much clearer name - changed

Reply:
In that case it would probably be worth making a similar change to `MFEMProblem::setMesh()` and `MFEMProblem::setDevice()` as well (plus any other similarly-named methods).